WordPress/Classes/Extensions/Colors and Styles/UIColor+MurielColors.swift

```diff
@@ -32,19 +32,39 @@ extension UIColor {
         return muriel(color: newColor)
     }
 }
+
 // MARK: - Basic Colors
 extension UIColor {
+
+    /// - Note: as part of this task: https://jetpackmobile.wordpress.com/2023/03/10/trial-task-user-defined-app-color/
```

Comment:
Hey, @tadija 👋 Since this is an open-source project, we do not include such internal links in commit messages, code, or comments directly.

CC: @wargcm

Reply:
Hey, thank you! 😊